src/main/java/com/gorlah/kappabot/rest/model/CommandRequest.java

```diff
@@ -1,10 +1,11 @@
 package com.gorlah.kappabot.rest.model;
 
-import lombok.Data;
+import com.gorlah.kappabot.function.BotRequestMetadata;
+import lombok.Value;
 
-@Data
-public class CommandRequest {
+@Value
+public class CommandRequest implements BotRequestMetadata {
```

Comment:
@ploober fyi, I changed these names on you so that simply adding the lombok @Value tag satisfies the new `BotRequestMetadata` interface.